docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -125,6 +125,13 @@ When the query results comparison is enabled, the query-tee compares the respons
 > If either Mimir cluster is running with a non-default value of `-ruler.evaluation-delay-duration`, we recommend setting `-proxy.compare-skip-recent-samples` to 1 minute more than the
 > value of `-ruler.evaluation-delay-duration`.
 
+### Slow query log
+
+The query-tee can optionally log requests that take longer on one backend than it did compared to the fastest backend.
+The logging is enabled by setting the flag `proxy.log-slow-query-response-threshold` to a duration whereby if two backends differ by the specified amount the query is logged.
+
+> **Note**: Slow query logging it disabled by default (by setting to 0).
```

Comment:
It looks like the default is actually `10s` (https://github.com/grafana/mimir/pull/7346/files#diff-ebfe64e8ba7589a6abc0cd137df8a1787800807b8e1fbead72ce78c62b194a67R55).

I would say that disabling the slow query log should not be a note but instead core to the documentation.

```suggestion
You can configure query-tee to log requests that take longer than the fastest backend by by setting the flag `proxy.log-slow-query-response-threshold`.

The default value is `10s` which logs requests that are ten seconds slower than the fastest backend.

To disable slow query logging, set `proxy.log-slow-query-response-threshold` to `0`.
```

Reply:
Thanks Jack! :-)

Reply:
Happy to help :)